contracts/src/registry/BaseRegistry.sol

```diff
@@ -74,7 +74,7 @@ abstract contract BaseRegistry is IRegistry, ERC1155Singleton {
      * @return The address of the registry for this subdomain, or `address(0)` if none exists.
      */
     function getSubregistry(string calldata label) external view virtual returns (IRegistry) {
-        (address subregistry,) = datastore.getSubregistry(uint256(keccak256(bytes(label))));
+        (address subregistry,) = datastore.getSubregistry(_labelToTokenId(label));
```

Comment:
I saw a library function for this in one of your other PRs - can we use that?

Reply:
Was going to save that for a merge, but sure will add it in now